service/src/main/java/tds/config/endpoints/ConfigController.java

```diff
@@ -0,0 +1,48 @@
+package tds.config.endpoints;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import tds.config.ClientSystemFlag;
+import tds.config.ClientTestProperty;
+import tds.config.services.ConfigService;
+import tds.config.web.resources.ClientSystemFlagResource;
+import tds.config.web.resources.ClientTestPropertyResource;
+
+
+@RestController
+@RequestMapping("/config")
+public class ConfigController {
+    private static final Logger LOG = LoggerFactory.getLogger(ConfigController.class);
```

Comment:
@millerts - I don't think we have this in the Exam Service.  We should standardize for sure, but does this also fall into the Monitoring and central logging which I believe is another story.